docs/usage.md

```diff
@@ -406,3 +407,74 @@ some database changes compared to the previous release.
 If you run into database migration warnings during the server start please
 check our [database schema upgrade guide's](db_schema_guide.md)
 `Database upgrade for running servers` section.
+
+# <a name="unique-report-indentifier"></a> Unique Report Identifier (RI)
+
+Each report has a unique (hash) identifier generated from checker name
+and the location of the finding: column number, textual content of the line,
+enclosing scope of the bug location (function signature, class, namespace).
+ 
+# <a name="listing-reports"></a> Listing and Counting Reports
+
+## How reports are counted?
+
+You can list analysis reports in two ways:
+1. Using the **`CodeChecker parse`** command, which **does not do deduplication**.
+2. Reports view of the **Web UI**, which **does deduplication**.
+
+These two views may show slightly different report list and counts based on how 
+duplicate findings or findings with the same hash identifier are rendered.
+
+The `CodeChecker parse` command does not do deduplication. 
+It lists reports simply as found by the
+analyzers and always lists all duplicate and similar findings.
+  
+You may find the same bug report multiple times for two reasons:
+1) The same source file is analyzed 2 times 
+(because the `compile_commmands.json` contains the build command 2 times)
+then the same findings will be listed 2 times. 
+2) All findings that are found in headers 
+will be shown as many times as many source file include that header.
+
+Web UI reports view on the other hand does deduplication: It will not show
+the same bug report two times even if the analyzer found it multiple times.
+
+## Report Uniqueing
+
+There is an additional uniqueing functionality in the 
+Web UI that helps the grouping findings that have the same 
+*Report Identifier* within or accross muliple runs.
+You can enable this functionality by ticking in the "Unique reports" tick box
+in the Bug Overview tab.
+
+This feature is useful when 
+* you want to list unique findings accross multiple
+runs. In this mode the same report stored in different runs is shown only once.
+* you want count reports as one which end up in the same same bug location, but
+reached through different paths. For example the same null pointer deference 
+error may occur on multiple execution paths.
+
+The **checker statistics** view shows an aggregate count of the reports accross
+multiple runs. The report counts shown on that page are calculated using the
+unique report identifiers.
+
+## How diffs between runs are calculated?
+
+Diffs between runs are calculated based on the Unique Report Identifier.
+
+Lets take run *A* and run *B* and take the diff between run *A* and *B*, 
+where *A* is the baseline.
+
+The base of the comparison are the reports that are not in 
+*detection status* "Resolved" and not in *review status* "False Positive".
```

Comment:
*detection status* `Resolved` and not in *review status* `False Positive` and `Intentional`